### ISSUE TITLE
Dev: WAVE Accessibility: Security Page

### ISSUE THREAD

**Issue by eunicode** [hackforla/tdm-calculator#2802]
### Overview

Related to audit #2709 

A WAVE audit of the Security page (`/roles`) returns these results:

Errors:
- Missing form label
- Broken ARIA reference

Alerts: 
- Orphaned form label
- Missing fieldset
- Noscript element
- Underlined text
- Layout table

### Action Items

- [x] Fix errors
- [x] Fix alerts when possible

More
- [ ] Review with product and dev. Update if needed based on feedback
- [ ] Once finalized, add before and after images to the slide in the staging deck for this issue (see link under resources)
- [ ] Get Stakeholder sign-off via the stakeholder meeting slide deck.

### Resources/Instructions

Accessibility audit spreadsheet: 
https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - [Staging slide, ISSUE NUMBER]
- Release Deck
   - [2025-MM-DD Release deck, slide ISSUE NUMBER]
